src/uu/du/src/du.rs

```diff
@@ -721,6 +721,12 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         excludes: build_exclude_patterns(&matches)?,
     };
 
+    let time_format = if let Some(_time) = time {
```

Comment:
```suggestion
    let time_format = if time.is_some() {
```
Other than that and the missing test, looks good :D

Reply:
Done